api/v1_claim_rewards.go

```diff
@@ -335,22 +427,21 @@ func sendRewardClaimTransactions(
 
 	// If no attestations need to be submitted, don't need to split into two txs
 	if len(attestations) > 0 {
-		preTx := tx
-		preTx.WithOpt(solana.TransactionAddressTables(addressLookupTables))
-		err = transactionSender.AddPriorityFees(ctx, preTx, spl.AddPriorityFeesParams{Percentile: 99, Multiplier: 1})
+		partialTx.WithOpt(solana.TransactionAddressTables(addressLookupTables))
+		err = transactionSender.AddPriorityFees(ctx, partialTx, spl.AddPriorityFeesParams{Percentile: 99, Multiplier: 1})
 		if err != nil {
 			return nil, err
 		}
-		err = transactionSender.AddComputeBudgetLimit(ctx, preTx, spl.AddComputeBudgetLimitParams{Padding: 1000, Multiplier: 1.2})
```

Comment:
so is the bug that the `tx` is _also_ getting this compute budget limit, because the builder is a pointer and they both point to the same memory location? whoops. probably should have cloned! nice fix

Reply:
yeah that's right. going into the simulation to create the new compute budget, we already have an instruction for compute budget which causes the simulation to fail

Reply:
yeah I think I expected the `preTx := tx` to be a copy, but it was a ref/ptr, yarg.